SciGraph-core/src/test/java/io/scigraph/internal/TinkerGraphUtilTest.java

```diff
@@ -175,7 +189,8 @@ public void graphsAreMerged() {
     Vertex g2v1 = graph2.addVertex(1);
     Vertex g2v2 = graph2.addVertex(2);
     Edge g2e1 = graph1.addEdge(1, g2v1, g2v2, "test2");
-    Graph graph = TinkerGraphUtil.combineGraphs(graph1, graph2);
+    TinkerGraphUtil tgu = new TinkerGraphUtil(graph1, curieUtil);
+    Graph graph = tgu.combineGraphs(graph2);
```

Comment:
Let's add a check that `tgu.getGraph()` returns the same graph as this `graph` object.